code/part-one/README.md

```diff
@@ -172,12 +174,13 @@ hash from your first block, and generate a new hash. Rinse and repeat.
 ]
 ```
 
-Now all of our data is linked to our previous data, all the way back to our
-original "genesis block", which is the only block without a previous hash.
-Anyone attempting to modify one block, would also have to modify _every_ block
-that came afterward it in the chain.
+Now all of our data is linked all the way back to our original genesis block.
```

Comment:
**Add punctuation:** Now`,`

Reply:
Maybe I'm wrong, but I feel like this changes the meaning. It would make "Now" a little introductory clause like "So," or "Anyway,". But the word actually helps define the remainder of the sentence. Now is _when_ all of our data is linked.